faucet.config.js

```diff
@@ -1,46 +1,28 @@
-let jsConfig = {
-  manifest: {
-    file: "public/assets/manifests/javascript.json",
-    baseURI: (bundlePath, baseName) => `/assets/javascripts/${baseName}`
-  },
-  bundles: [{
-    entryPoint: "./app/assets/javascripts/application.js",
-    target: "public/assets/javascripts/application.js",
-    externals: {}
-  }]
-};
+let path = require("path");
 
-let sassConfig = {
-  manifest: {
-    file: "public/assets/manifests/stylesheet.json",
-    baseURI: (bundlePath, baseName) => `/assets/stylesheets/${baseName}`
-  },
-  assets: [
-    "public/assets/manifests/static.json"
-  ],
-  prefixes: {
-    browsers: [ "last 2 versions" ]
-  },
-  bundles: [{
-    entryPoint: "app/assets/stylesheets/application.scss",
-    target: "public/assets/stylesheets/application.css"
-  }]
-};
+let jsConfig = [{
+  source: "./app/assets/javascripts/application.js",
+  target: "./public/assets/javascripts/application.js"
+}];
 
-let staticConfig = {
-  manifest: {
-    file: "public/assets/manifests/static.json",
-    baseURI: (bundlePath, baseName) => `/assets/static/${baseName}`
-  },
-  bundles: [{
-    source: "app/assets/images",
-    target: "public/assets/static"
-  }]
-}
+let sassConfig = [{
+  source: "./app/assets/stylesheets/application.scss",
+  target: "./public/assets/stylesheets/application.css"
+}];
+
+let staticConfig = [{
+  source: "./app/assets/images",
+  target: "./public/assets/images"
+}];
 
 module.exports = {
   js: jsConfig,
   sass: sassConfig,
   static: staticConfig,
+  manifest: {
+    file: "./public/assets/manifest.json",
+    key: (f, targetDir) => path.relative(targetDir, f),
+    value: f => `/${path.relative("public", f)}`
+  },
   watchDirs: ["app/assets"]
```

Comment:
FWIW, I like to keep `watchDirs` at the very top of the config object.

I'm also not sure whether it's still worth using separate variables here; I'd just inline `*Config`.

Reply:
Done. Except the watchDirs. Because I don't know why 🤔